### ISSUE TITLE
Can't add additional season when only one season is already added

### ISSUE THREAD

**Issue by dataprolet** [toddrob99/searcharr#97]
When I add season 01 of a show via the bot and want to add season 02 later, it says "Already added", even though only the one season is present.
Is this intended?

**Comment by toddrob99** [comment]
Yes, this is how it's designed currently. The ability to add individual seasons was added, but only initially. The bot doesn't currently check if the show is added with only certain seasons, only if it's added at all.
